app/models/outcome_code.rb

```diff
@@ -28,6 +28,10 @@ def self.all
   end
   # rubocop:enable Metrics/AbcSize, Metrics/MethodLength
 
+  def self.name_by_id(outcome_id)
+    @all.first { |item| item.id == outcome_id }.name
```

Comment:
can we change this to 
```
@all.first { |item| item.id == outcome_id }.description
```

as otherwise we are forcing the ID in the return value. :)